src/navigation/constants.js

```diff
@@ -33,4 +33,6 @@ export const ROUTES = {
   STOCKTAKE_MANAGER: 'stocktakeManager',
   STOCKTAKE_EDITOR: 'stocktakeEditor',
   STOCKTAKE_EDITOR_WITH_REASONS: 'stocktakeEditorWithReasons',
+
```

Comment:
I appreciate what the new lines in this object code is doing, but it's new to me and looks weird.

Reply:
its good aye

Reply:
Dunno if I agree with trying to do with props destructuring, which is what I think you were doing in https://github.com/openmsupply/mobile/pull/1522#discussion_r344989621

Reply:
nah that was just a mistake sorry